src/lib/formats/tzx_cas.cpp

```diff
@@ -200,9 +205,9 @@ static inline int millisec_to_samplecount( int millisec )
 	return (int) (millisec * ((double)TZX_WAV_FREQUENCY / 1000.0));
 }
 
-static inline int tcycles_to_samplecount( int tcycles )
+static inline int tcycles_to_samplecount( int tcycles, float t_scale )
```

Comment:
Is there any point making `t_scale` a `float` when it’s always converted to `double` on use?

Reply:
I put it like this because the variable was created as float. I correct it